src/operator/operator_common.h

```diff
@@ -309,6 +309,34 @@ inline void ParamParser(nnvm::NodeAttrs* attrs) {
   attrs->parsed = std::move(param);
 }
 
+/*! \brief Callback class to allow for convenient development and testing */
+template<typename Type>
+class Callbacker {
```

Comment:
For unit testing callback into the unit testing framework. I can move it to just batchnorm if you like.